packages/sdks/web-js-sdk/src/enhancers/helpers/index.ts

```diff
@@ -116,11 +126,16 @@ export const getUserFromResponse = async (
 export const isDescopeBridge = () =>
   typeof window !== 'undefined' && !!window['descopeBridge'];
 
-export const isLocalStorage = typeof localStorage !== 'undefined';
+export const isLocalStorage =
+  typeof customStorage !== 'undefined' || typeof localStorage !== 'undefined';
```

Comment:
just double checking - if custom storage was provided, we want to expire nonce?
<img width="1125" height="409" alt="image" src="https://github.com/user-attachments/assets/10f6a134-1285-4618-baae-818c2712f14d" />

Reply:
According to the code, if there is no storage, we skip nonce cleanup
I think that's ok